packages/meditrak-server/src/tests/routes/userAccounts/EditUserAccounts.test.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { expect } from 'chai';
+import { findOrCreateDummyRecord, findOrCreateDummyCountryEntity } from '@tupaia/database';
+import { Authenticator } from '@tupaia/auth';
+import {
+  TUPAIA_ADMIN_PANEL_PERMISSION_GROUP,
+  BES_ADMIN_PERMISSION_GROUP,
+} from '../../../permissions';
+import { TestableApp } from '../../TestableApp';
+import { prepareStubAndAuthenticate } from '../utilities/prepareStubAndAuthenticate';
+
+describe('Permissions checker for EditUserAccounts', async () => {
+  const DEFAULT_POLICY = {
+    DL: ['Public'],
+    KI: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Admin'],
+    SB: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Royal Australasian College of Surgeons'],
+    VU: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Admin'],
+    LA: ['Admin'],
+    TO: ['Admin'],
+  };
+
+  const BES_ADMIN_POLICY = {
+    LA: [BES_ADMIN_PERMISSION_GROUP],
+  };
+
+  const app = new TestableApp();
+  const { models } = app;
+  let userAccount1;
+  let userAccount2;
+
+  before(async () => {
+    const permissionGroup = await findOrCreateDummyRecord(models.permissionGroup, {
```

Comment:
```suggestion
    const publicPermissionGroup = await findOrCreateDummyRecord(models.permissionGroup, {
```
Makes it easier to read the creation of the user entity permission records